bundles/org.openhab.ui/web/src/components/thing/channel-link.vue

```diff
@@ -44,6 +44,7 @@
       <f7-icon slot="media" color="green" aurora="f7:plus_circle_fill" ios="f7:plus_circle_fill" md="material:control_point" />
     </f7-list-item>
     <f7-list-button class="searchbar-ignore" color="blue" :title="(channelType.parameterGroups.length || channelType.parameters.length) ? 'Configure Channel' : 'Channel Details'" @click="configureChannel()" />
+    <f7-list-button class="searchbar-ignore" v-if="extensible" color="blue" :title="(channelType.parameterGroups.length || channelType.parameters.length) ? 'Copy Channel' : ''" @click="copyChannel()" />
```

Comment:
```suggestion
    <f7-list-button class="searchbar-ignore" v-if="extensible && thing.editable" color="blue" :title="(channelType.parameterGroups.length || channelType.parameters.length) ? 'Copy Channel' : ''" @click="copyChannel()" />
```

I just noticed that the remove channel button is visible for file-provisioned Things, which shouldn’t be the case because they are not editable. 

Can you please add the suggested condition to copy channel and remove channel?

In general the implementation of the channel management lacks some handling for not-editable Things …

Reply:
done